### DISCUSSION TITLE
Code Quality Tools: Open Discussion

### DISCUSSION THREAD

**Discussion by kmesiab** [kmesiab/orion#7] [General]
# SECURITY!

Since we work with 🔑 API keys, it's important to take steps to ensure security and integrity of our repo.  

I highly encourage installing 🛡️ [GitGuardian](https://dashboard.gitguardian.com/workspace/485251/get-started) as a pre-commit hook.  We will be enabling it on the repository, however, if a secret is caught in the build pipeline, it is too late to recover it, and you may not be in a position to rewrite the commit history!

https://dashboard.gitguardian.com/workspace/485251/get-started

🍻

**Comment by frankie-mur** [comment]
Great tip! Was not aware GitGuardian could be installed locally as a pre-commit hook very cool! 

> # SECURITY!
> Since we work with 🔑 API keys, it's important to take steps to ensure security and integrity of our repo.
> 
> I highly encourage installing 🛡️ [GitGuardian](https://dashboard.gitguardian.com/workspace/485251/get-started) as a pre-commit hook. We will be enabling it on the repository, however, if a secret is caught in the build pipeline, it is too late to recover it, and you may not be in a position to rewrite the commit history!
> 
> https://dashboard.gitguardian.com/workspace/485251/get-started
> 
> 🍻
